src/CopyToClipboard/CopyToClipboard.component.tsx

```diff
@@ -74,7 +74,7 @@ const SCopyToClipboard = styled.div`
   display: ${(props: Props) => (props.fullWidth ? 'flex' : 'inline-flex')};
   padding: ${(props: Props) => props.theme.copyToClipboard.padding};
   background-color: ${(props: Props) =>
-    props.background!.toString() || props.theme.colors[props.background!]};
+    props.theme.colors[props.background!] || props.background!.toString()};
```

Comment:
`props.background` should be evaluated first, but it looks like the `background: 'disabledBackground'` in `defaultProps` was causing `props.theme.colors[props.background!]` to be unreachable.

In the `defaultProps` object if we change `background`'s value to `''` , then we can keep it like `props.background || props.theme.colors[props.background!]` so that the incoming prop can override the theme.

The same treatment should apply to other components as well.

@ben-dalton

Reply:
@dorokhovich-mikhail Along with Tamer's comments, something I forgot to mention - we had taken an approach in some of the components of taking the prop input of a color and mapping it to a color found in the theme file. It's a nice idea, but not that intuitive. We'll want to convert that over to using the exact value passed in to the prop. So, instead of `props.theme.colors[props.background!]`, there should be a `background` value in the component section of theme for the background. When we set the background style it should be `props.background || props.theme.copyToClipboard.background]`. We currently have two themes so we'll need to update both themes if a value is added.